src/Pages/LandingPage.js

```diff
@@ -48,11 +59,50 @@ const LandingPageObserver = observer(
           >
             {articles.map((article, index) => {
               if (index === 3) {
-                return <LargeCardPost article={article} />;
+                return (
+                  <div
```

Comment:
Should these just be React Router `Link`s? It's not significant at the moment, just curious.

Reply:
The Link element is causing the text inside the card to be converted to hyperlinks.
We can use CSS.
I chose to use programmatic navigation, hence the onCLick events.